### ISSUE TITLE
feature: progress bar for `kit s` over SSH

### ISSUE THREAD

**Issue by 0x70b1a5** [kinode-dao/kit#279]
**Is your feature request related to a problem? Please describe.**
On slow connections,`kit s` can take a while to upload large packages. Sometimes it's unclear why the command takes so long to complete.

**Describe the solution you'd like**
An upload progress bar would unambiguously indicate that bandwidth was the blocker.

**Describe alternatives you've considered**
Waiting :)

**Comment by nick1udwig** [comment]
Good idea. Looks like there are some good options out there such as indicatif, progress-bar, and pbr. I'll think about where else we could add useful indicators like this.
